cve_bin_tool/data_sources/epss_source.py

```diff
@@ -118,5 +118,5 @@ def parse_epss_data(self, file_path=None):
         # Parse the data from the remaining rows
         for row in reader:
             cve_id, epss_score, epss_percentile = row[:3]
-            parsed_data.append((cve_id, "EPSS", epss_score, epss_percentile))
+            parsed_data.append((cve_id, 1, epss_score, epss_percentile))
```

Comment:
@Rexbeast2 You should find the value of the EPSS metric by searching for the EPSS string in the metrics table rather than assume it is '1'